templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Delete Manufacturer</h1>
+    <p>Are you sure you want to delete the manufacturer: {{ manufacturer }}?</p>
+    <p><i>All Cars with this manufacturer will be also deleted!</i></p>
+
+    <form action="" method="POST">
```

Comment:
It's not necessary, but I would refer to write method in all forms in one way: upper or lower

Reply:
I don’t see a problem here, just like Danya showed in the video

Reply:
Yes for sure, it's absolutely not a problem, just my own point of view